site/getting-started.md

```diff
@@ -57,7 +60,21 @@ Deploying Contour using either option creates:
 - A Service of `type: LoadBalancer` that points to the Contour's Envoy instances
 - Depending on your deployment environment, new cloud resources -- for example, a cloud load balancer
 
-### Example workload
+### Option 2: Install using Helm
```

Comment:
This should be 'Option 3: Install using Helm' now, could you also please add "install using operator" to Option 2 above?

Reply:
Updated in the latest commit.